src/sections/2021/section-2.js

```diff
@@ -0,0 +1,176 @@
+import React from "react"
+import SectionHeader from '../../components/2021/section-header'
+import Grid from '../../components/2021/grid'
+import GridCell from '../../components/2021/grid-cell'
+import PieChart from '../../components/2021/pie-chart'
+import BarChart from '../../components/2021/bar-chart'
+import Quote from '../../components/2021/quote'
+import Figure from "../../components/2021/figure"
+import SmallCallout from "../../components/2021/small-callout"
+
+const Section2 = () => (
+    <Grid>
+      <GridCell spanMD="8" spanLG="12">
+        <SectionHeader title="Design System & Their Teams" number="2" total="6" />
```

Comment:
The title should be "The Respondents"

Reply:
Good catch, but “The Respondents” is for section 1. This should be “Characteristics and Teams”

Reply:
Ah, yes!